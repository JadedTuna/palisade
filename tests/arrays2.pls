in {
    high arg: bool;
}
out {
    low arr[3]: bool;
}

x[3] := [false, true, arg];
// error: size mismatch
y[4] := arr;
