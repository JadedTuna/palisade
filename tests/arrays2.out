[1;31merror: [0msize mismatch between arrays
   8 | x[3] := [false, true, arg];
   9 | // error: size mismatch
  10 | y[4] := [1;31marr[0m;
       [1;31m~~~~~~~~[0m[1;31m^^^[0m
